### ISSUE TITLE
Failed submission notifications

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/va.gov-team#41579]

**Comment by michelpmcdonald** [comment]
The fix to send notifications to users who hit the "PIF in Use" error when attempting to submit a 526 claim has been merged.
However, claim failures with a "PIF in Use" error message have gone away due to an attempted BGS(in their codebase, vets-api calls EVSS, EVSS calss BGS).

That being said, it looks like only the actual "PIF in Use" error message has gone away, it looks like the error message has been changed to the much more generic "External Service Error".

However, "External Service Error" is a very generic error that has other causes other than PIF in use.  To try to figure out what the specific issue is for claims that fail out with "External Service Error", we have to reach out on a claim by claim basis to EVSS, they investigate, and provide the underlying error.  Based on these interactions with EVSS, a lot of these "external Service Error" claims have an underlying error of "Open Work Item".  However, since EVSS can only provide the underlying "open work item" error message manually via slack\emails, my vets-api code can't act on them

TLDR; Closing this issue, because the work was complete, but we can no longer identify "PIF in Use" submission errors in vets-api due to downstream changes in BGS so we can no longer send the "Pif In Use" error notification to the vets.
